cmd/process-agent/main_common.go

```diff
@@ -181,7 +181,7 @@ func runAgent(exit chan struct{}) {
 		}()
 	}
 
-	cfg, err := config.NewAgentConfig(loggerName, opts.configPath, opts.sysProbeConfigPath)
+	cfg, err := config.NewAgentConfig(loggerName, opts.configPath, opts.sysProbeConfigPath, true)
```

Comment:
Aren't we changing the current behavior by assuming that we're always able to collect containers?
Should we use something like?
```golang
	// Note: This only considers container sources that are already setup. It's possible that container sources may
	//       need a few minutes to be ready on newly provisioned hosts.
	_, err = util.GetContainers()
	canAccessContainers := err == nil
    cfg, err := config.NewAgentConfig(loggerName, opts.configPath, opts.sysProbeConfigPath, canAccessContainers)
```

And then during tests we can manually set it o `true` or `false` according to the behavior we're expecting.

Reply:
Oh that's a good point. I will do that :)